### DISCUSSION TITLE
OAuth - Azure AD SSO

### DISCUSSION THREAD

**Discussion by nickdring** [joomla/joomla-cms#38339] [General] [answered]
Hi guys, I know there is a guide here for SSO. 
https://docs.joomla.org/Azure_AD_SSO_Single_Sign_on_Joomla_using_Oauth_Client_Openid_Connect f
But this isn't a core plugin, it's from a third-party developer.
The basic plugin is free version but has basic functionality as you can see here https://plugins.miniorange.com/joomla-ldap#pricing 

My question is as more companies that use Joomla move away from old authentication protocols such as LDAP to cloud-based alternatives such as AAD I wonder if there are any plans to add this functionality to the core?

Cheers

**Comment by rdeutz** [comment, marked as answer]
No plans to add something like this to the core I am aware of. I see it more as a 3rd part extension

**Reply by nickdring** [reply]
I understand, however as time goes by though more and more companies and institutions are ditching their internal AD and moving to the cloud with AAD and others. We are one such institution. At the moment if we want to use AAD we have to use the Mini orange plugin. We have around 80+ sites. So as you can imagine the cost to buy 80+ licenses is not cheap.
